runtime-modules/storage/src/tests/mock.rs

```diff
@@ -63,6 +63,7 @@ impl SetLeadFixture {
             role_account_id: DEFAULT_LEADER_ACCOUNT_ID,
             reward_relationship: None,
             role_stake_profile: None,
+            storage: Vec::new(),
```

Comment:
This field can also now be removed.

Reply:
https://github.com/Joystream/joystream/pull/2180/commits/dc39181954b99319c9fb98b0ff9cecb3ec475fe5